dotcom-rendering/src/lib/useAdBlockAsk.ts

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from 'react';
+import { useAB } from './useAB';
+
+const useIsInAdBlockAskVariant = (): boolean => {
+	const abTestAPI = useAB()?.api;
+	const isInVariant = !!abTestAPI?.isUserInVariant('AdBlockAsk', 'variant');
+	return isInVariant;
+};
+
+const detectByRequests = async () => {
```

Comment:
This detection function should get us started for a zero percent test, but we may want to refine it before we go-live!

Reply:
Might be worth adding a JSDoc to that effect?